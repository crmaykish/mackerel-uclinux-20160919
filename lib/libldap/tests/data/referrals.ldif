dn: c=US
c: US
objectclass: country

dn: o=ABC,c=US
o: ABC
ref: ldap://hostA/o=abc,c=us HostA
ref: ldap://hostB HostB
objectclass: referral
objectclass: extensibleObject

dn: o=XYZ,c=US
o: XYZ
ref: ldap://hostC/o=xyz,c=us HostC
objectclass: referral
objectclass: extensibleObject

dn: o=University of Michigan,c=US
o: University of Michigan
objectclass: organization

dn: cn=Manager,o=University of Michigan,c=US
cn: Manager
cn: Directory Manager
cn: Dir Man
sn: Manager
description: Manager of the directory
userpassword:: c2VjcmV0
objectclass: person
