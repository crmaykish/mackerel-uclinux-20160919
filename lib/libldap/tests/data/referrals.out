# extended LDIF
#
# LDAPv3
# base <c=US> with scope sub
# filter: (objectClass=referral)
# requesting: * ref 
# with manageDSAit critical control
#

# ABC, US
dn: o=ABC,c=US
o: ABC
ref: ldap://hostA/o=abc,c=us HostA
ref: ldap://hostB HostB
objectClass: referral
objectClass: extensibleObject

# XYZ, US
dn: o=XYZ,c=US
o: XYZ
ref: ldap://hostC/o=xyz,c=us HostC
objectClass: referral
objectClass: extensibleObject

# search result
search: 2
result: 0 Success

# numResponses: 3
# numEntries: 2
# extended LDIF
#
# LDAPv3
# base <o=abc,c=US> with scope sub
# filter: (objectClass=referral)
# requesting: * ref 
# with manageDSAit critical control
#

# ABC, US
dn: o=ABC,c=US
o: ABC
ref: ldap://hostA/o=abc,c=us HostA
ref: ldap://hostB HostB
objectClass: referral
objectClass: extensibleObject

# search result
search: 2
result: 0 Success

# numResponses: 2
# numEntries: 1
# extended LDIF
#
# LDAPv3
# base <uid=xxx,o=abc,c=US> with scope sub
# filter: (objectClass=referral)
# requesting: * ref 
# with manageDSAit critical control
#

# search result
search: 2
result: 10 Referral
matchedDN: o=ABC,c=US
ref: ldap://hostA/uid=xxx,o=abc,c=us??sub
ref: ldap://hostB/uid=xxx,o=abc,c=US??sub

# numResponses: 1
# extended LDIF
#
# LDAPv3
# base <c=US> with scope base
# filter: (objectclass=*)
# requesting: 1.1 
#

# US
dn: c=US

# search result
search: 2
result: 0 Success

# numResponses: 2
# numEntries: 1
# extended LDIF
#
# LDAPv3
# base <c=US> with scope one
# filter: (objectclass=*)
# requesting: 1.1 
#

# search reference
ref: ldap://hostA/o=abc,c=us??base
ref: ldap://hostB/o=ABC,c=US??base

# search reference
ref: ldap://hostC/o=xyz,c=us??base

# University of Michigan, US
dn: o=University of Michigan,c=US

# search result
search: 2
result: 0 Success

# numResponses: 4
# numEntries: 1
# numReferences: 2
# extended LDIF
#
# LDAPv3
# base <c=US> with scope sub
# filter: (objectclass=*)
# requesting: 1.1 
#

# search reference
ref: ldap://hostA/o=abc,c=us??sub
ref: ldap://hostB/o=ABC,c=US??sub

# search reference
ref: ldap://hostC/o=xyz,c=us??sub

# US
dn: c=US

# University of Michigan, US
dn: o=University of Michigan,c=US

# Manager, University of Michigan, US
dn: cn=Manager,o=University of Michigan,c=US

# search result
search: 2
result: 0 Success

# numResponses: 6
# numEntries: 3
# numReferences: 2
# extended LDIF
#
# LDAPv3
# base <o=abc,c=US> with scope base
# filter: (objectclass=*)
# requesting: 1.1 
#

# search result
search: 2
result: 10 Referral
matchedDN: o=ABC,c=US
ref: ldap://hostA/o=abc,c=us??base
ref: ldap://hostB/o=abc,c=US??base

# numResponses: 1
# extended LDIF
#
# LDAPv3
# base <o=abc,c=US> with scope one
# filter: (objectclass=*)
# requesting: 1.1 
#

# search result
search: 2
result: 10 Referral
matchedDN: o=ABC,c=US
ref: ldap://hostA/o=abc,c=us??one
ref: ldap://hostB/o=abc,c=US??one

# numResponses: 1
# extended LDIF
#
# LDAPv3
# base <o=abc,c=US> with scope sub
# filter: (objectclass=*)
# requesting: 1.1 
#

# search result
search: 2
result: 10 Referral
matchedDN: o=ABC,c=US
ref: ldap://hostA/o=abc,c=us??sub
ref: ldap://hostB/o=abc,c=US??sub

# numResponses: 1
# extended LDIF
#
# LDAPv3
# base <uid=xxx,o=abc,c=US> with scope base
# filter: (objectclass=*)
# requesting: 1.1 
#

# search result
search: 2
result: 10 Referral
matchedDN: o=ABC,c=US
ref: ldap://hostA/uid=xxx,o=abc,c=us??base
ref: ldap://hostB/uid=xxx,o=abc,c=US??base

# numResponses: 1
# extended LDIF
#
# LDAPv3
# base <uid=xxx,o=abc,c=US> with scope one
# filter: (objectclass=*)
# requesting: 1.1 
#

# search result
search: 2
result: 10 Referral
matchedDN: o=ABC,c=US
ref: ldap://hostA/uid=xxx,o=abc,c=us??one
ref: ldap://hostB/uid=xxx,o=abc,c=US??one

# numResponses: 1
# extended LDIF
#
# LDAPv3
# base <uid=xxx,o=abc,c=US> with scope sub
# filter: (objectclass=*)
# requesting: 1.1 
#

# search result
search: 2
result: 10 Referral
matchedDN: o=ABC,c=US
ref: ldap://hostA/uid=xxx,o=abc,c=us??sub
ref: ldap://hostB/uid=xxx,o=abc,c=US??sub

# numResponses: 1
